src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -179,16 +179,21 @@ export class Breadcrumbs extends React.Component<BreadcrumbsProps, BreadcrumbsSt
     }
 
     renderMoreItem() {
-        const {popupStyle, popupPlacement, renderItemDivider} = this.props;
         const {hiddenItems} = this.state;
+        if (hiddenItems.length === 0) {
+            return null;
+        }
```

Comment:
not sure that it should be lifted from component here

Reply:
Same here.

Reply:
I think about `BreadcrumbsMore` as composite component, than component to render specific dropdown.

I agree that it would be nice to have such component, if it was used somewhere else. But now it looks like more like refactor for refactor to me.